examples/allocator_access.cpp

```diff
@@ -0,0 +1,76 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include <string>
+
+#include "umpire/Allocator.hpp"
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Umpire.hpp"
+
+using cPlatform = camp::resources::Platform;
+
+bool testAccess(umpire::Allocator a)
+{
+  if(is_accessible(cPlatform::host, a)) {
```

Comment:
Shouldn't this call have a namespace qualifier?  I am curious why this compiles.  Is there a `using umpire` somewhere?  if so, that doesn't seem right to me.

```suggestion
  if(umpire::is_accessible(cPlatform::host, a)) {
```

Reply:
The is_accessible function is housed in the Umpire.hpp file, which is included. I assume that's why it doesn't complain of an error. I could try to do some kind of statement like `using access = <path to is_accessible function>` similar to how I did the `cPlatform` statement instead (I'd have to test it, haven't tried as of yet). Is one better than the other?